modules/standard/Map.chpl

```diff
@@ -398,18 +398,20 @@ module Map {
     }
 
     /*
-      Get the value mapped to the given key, or add the mapping if key does not
-      exist.
+      Get the value mapped to the given key if the key exists in the
+      map. If the key does not exist and the value type is default
+      initializable, the mapping is added. Otherwise, throws an error.
```

Comment:
Me again.  "If A and B, the mapping is added, Otherwise , throws an error".  Thus "If !A, throws an error".  But !A is "the key exists".  And I don't think "If the key exists, throws an error" is right.